### PR TITLE
updated free-programming-books-en.md

### PR DESCRIPTION
**Pull request by bytesofbec**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by shrn01 — COMMENTED** [review]
Please add the resources to the proper file. in this case free-programming-books-langs.md. Read docs/CONTRIBUTING.md for more information

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -502,16 +502,23 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [C++ GUI Programming With Qt 3](https://ptgmedia.pearsoncmg.com/images/0131240722/downloads/blanchette_book.pdf) - Jasmin Blanchette, Mark Summerfield (PDF)
 * [C++ Language](http://www.cplusplus.com/doc/tutorial/)
+* [C++ Language](https://cplusplus.com/doc/tutorial)
 * [C++ Notes for Professionals](https://goalkicker.com/CPlusPlusBook) - Compiled from StackOverflow Documentation (PDF)
 * [C++ Programming](https://en.wikibooks.org/wiki/C%2B%2B_Programming) - Panic, et al.
 * [C++ Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/cplusplus) (PDF, Kindle) (email address *requested*, not required)
 * [C++ Tricks](http://www.bordoon.com/cplusplus/book_wrapper.html)
+* [C++ Tutorial](https://www.w3schools.com/cpp)
+* [C++ Tutorial](https://www.cprogramming.com/tutorial/c++-tutorial.html)
+* [C++ Tutorial for Beginners - Full Course](https://youtu.be/vLnPwxZdW4Y)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:520]: Move to courses file

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -502,16 +502,23 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
+* [C++ Tutorial for Beginners - Learn C++ in 1 Hour](https://youtu.be/ZzaPdXTrSb8)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:521]: Move to courses file

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -1130,6 +1137,9 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [JavaScript Spessore](https://web.archive.org/web/20160325064800/https://leanpub.com/javascript-spessore/read) - Reginald Braithwaite (HTML)
 * [JavaScript Succinctly](https://www.syncfusion.com/resources/techportal/ebooks/javascript) - Cody Lindley (PDF, Kindle; email address *requested*, not required)
 * [JavaScript the Right Way](https://github.com/braziljs/js-the-right-way) - William Oliveira, Allan Esquina (HTML)
+* [Javascript Tutorial](https://www.w3schools.com/js)
+* [Javascript Tutorial](https://www.tutorialspoint.com/javascript/index.htm)
+* [JavaScript Tutorial for Beginners - Full Course in 8 Hours [2020]](https://youtu.be/Qqx_wzMmFeA)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:1142]: Move to courses file

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -1130,6 +1137,9 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [JavaScript Spessore](https://web.archive.org/web/20160325064800/https://leanpub.com/javascript-spessore/read) - Reginald Braithwaite (HTML)
 * [JavaScript Succinctly](https://www.syncfusion.com/resources/techportal/ebooks/javascript) - Cody Lindley (PDF, Kindle; email address *requested*, not required)
 * [JavaScript the Right Way](https://github.com/braziljs/js-the-right-way) - William Oliveira, Allan Esquina (HTML)
+* [Javascript Tutorial](https://www.w3schools.com/js)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:1140]: We can move this to interactive tutorials as the user can interact with the code and run it

File: books/free-programming-books-langs.md [hunk 5]
```diff
@@ -502,16 +502,23 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md) - `edt.:` Bjarne Stroustrup, Herb Sutter
 * [C++ For Programmers](https://tfetimes.com/wp-content/uploads/2015/04/c-for-c-programmers.pdf) - JT Kalnay (PDF)
 * [C++ GUI Programming With Qt 3](https://ptgmedia.pearsoncmg.com/images/0131240722/downloads/blanchette_book.pdf) - Jasmin Blanchette, Mark Summerfield (PDF)
 * [C++ Language](http://www.cplusplus.com/doc/tutorial/)
+* [C++ Language](https://cplusplus.com/doc/tutorial)
 * [C++ Notes for Professionals](https://goalkicker.com/CPlusPlusBook) - Compiled from StackOverflow Documentation (PDF)
 * [C++ Programming](https://en.wikibooks.org/wiki/C%2B%2B_Programming) - Panic, et al.
 * [C++ Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/cplusplus) (PDF, Kindle) (email address *requested*, not required)
 * [C++ Tricks](http://www.bordoon.com/cplusplus/book_wrapper.html)
+* [C++ Tutorial](https://www.w3schools.com/cpp)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:518]: We can move this to interactive tutorials as the user can interact with the code and run it

**Review by shrn01 — COMMENTED** [review]
Please make the suggested changes

File: books/free-programming-books-langs.md (hunk 5 above)
Comment (eshellman) [reply books/free-programming-books-langs.md:518]:
```suggestion
```
already there

File: books/free-programming-books-langs.md (hunk 4 above)
Comment (eshellman) [reply books/free-programming-books-langs.md:1139]:
```suggestion
```
already there

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (eshellman) [reply books/free-programming-books-langs.md:519]:
```suggestion
```
adding in #9203

File: books/free-programming-books-langs.md [hunk 6]
```diff
@@ -503,15 +503,18 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### <a id="cpp"></a>C++
 
 * [A Complete Guide to Standard C++ Algorithms](https://github.com/HappyCerberus/book-cpp-algorithms) - Šimon Tóth (PDF, LaTeX) (:construction: *in process*)
+* [C/C++ Basics](http://www.icce.rug.nl/documents/cplusplus) - Frank B. Brokken
```
Comment (eshellman) [inline books/free-programming-books-langs.md:506]:
```suggestion
```
duplicate

File: books/free-programming-books-langs.md [hunk 7]
```diff
@@ -502,16 +502,20 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 
+* [A Complete Guide to Programming in C++](http://www.lmpt.univ-tours.fr/~volkov/C++.pdf) - Ulla Kirch-Prinz (PDF)
 * [A Complete Guide to Standard C++ Algorithms](https://github.com/HappyCerberus/book-cpp-algorithms) - Šimon Tóth (PDF, LaTeX) (:construction: *in process*)
+* [C/C++ Basics](http://www.icce.rug.nl/documents/cplusplus) - Frank B. Brokken
 * [C++ Annotations](https://fbb-git.gitlab.io/cppannotations) - Frank B. Brokken (HTML, PDF)
 * [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md) - `edt.:` Bjarne Stroustrup, Herb Sutter
 * [C++ For Programmers](https://tfetimes.com/wp-content/uploads/2015/04/c-for-c-programmers.pdf) - JT Kalnay (PDF)
 * [C++ GUI Programming With Qt 3](https://ptgmedia.pearsoncmg.com/images/0131240722/downloads/blanchette_book.pdf) - Jasmin Blanchette, Mark Summerfield (PDF)
 * [C++ Language](http://www.cplusplus.com/doc/tutorial/)
+* [C++ Language](https://cplusplus.com/doc/tutorial)
```
Comment (eshellman) [inline books/free-programming-books-langs.md]:
```suggestion
* [C++ Language](https://cplusplus.com/doc/tutorial) - Alex Allain
```

File: books/free-programming-books-langs.md [hunk 8]
```diff
@@ -512,6 +512,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [C++ Programming](https://en.wikibooks.org/wiki/C%2B%2B_Programming) - Panic, et al.
 * [C++ Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/cplusplus) (PDF, Kindle) (email address *requested*, not required)
 * [C++ Tricks](http://www.bordoon.com/cplusplus/book_wrapper.html)
+* [C++ Tutorial](https://www.cprogramming.com/tutorial/c++-tutorial.html)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:515]:
```suggestion
* [C++ Tutorial](https://www.cprogramming.com/tutorial/c++-tutorial.html) - Alex Allain
```

**Pull request merged**